app/_hub/moesif/kong-plugin-moesif/index.md

```diff
@@ -1,37 +1,49 @@
 ---
-name: Moesif API Insights
+name: Moesif API Analytics
 publisher: Moesif
 
-categories:
+categories: 
   - analytics-monitoring
+  - logging
 
 type: plugin
 
-desc: User-centric API analytics and monitoring  
+desc: User-centric API analytics and logging  
 
 description: |
-  Monitor API traffic and understand usage with [Moesif's](https://www.moesif.com/solutions/track-api-program?language=kong-api-gateway) user-centric 
-  API analytics, including:
+  Monitor API traffic and analyze customer usage in [Moesif's API Analytics](https://www.moesif.com/solutions/track-api-program?language=kong-api-gateway&utm_medium=docs&utm_campaign=partners&utm_source=kong) platform, enabling:
```

Comment:
"enabling" on its own isn't a grammatically correct lead-in to the list items below. Suggest:

```suggestion
  Monitor API traffic and analyze customer usage in [Moesif's API Analytics](https://www.moesif.com/solutions/track-api-program?language=kong-api-gateway&utm_medium=docs&utm_campaign=partners&utm_source=kong) platform, which enables you to:
```

The lead-in should work as a sentence with each list item.